analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -632,6 +602,37 @@ def determine_compiler(gcc_command, is_executable_compiler_fun):
     return gcc_command[0]
 
 
+def filter_compiler_includes(include_dirs):
```

Comment:
Why was this function moved here? It is the same module just the name was changed.

Reply:
This function was a method of ICI. This class determined the compiler includes only once per compiler and language because the result is cached in the class. If I want to test the include path determination with and without filtering them then this function should filter the paths after caching. I moved this function out of the file so I can call it in `parse_options()` as a helper.